src/resources/js/views/month-mobile-events.js

```diff
@@ -69,12 +74,55 @@ tribe.events.views.monthMobileEvents = {};
 			if ( contentId ) {
 				$content = $container.find( '#' + contentId );
 				tribe.events.views.accordion.closeAccordion( $header, $content );
+				hasDays = true;
 			}
 
 			obj.closeMobileEvents( $header, $content );
 		} );
 	};
 
+	/**
+	 * Handles the user selection of a day in the Grid.
+	 *
+	 * @since TBD
+	 *
+	 * @param {bool} viewHasDays Whether the day selected by the user has days in it or not.
+	 *
+	 * @return {void} The method does not return a value and will hide and show message elements.
+	 */
+	obj.handleMobileDayClick = function( hasDays ) {
+		var $defaultNotices = $( obj.selectors.mobileEventsDefaultNotices );
+		var $dynamicNotices = $( obj.selectors.mobileEventsDynamicNotices );
```

Comment:
Everything we do in the JS should be scoped to the `$container`, so instead, we call something like `$container.find( obj.selectors.mobileEventsDefaultNotices )`. If this is being called from another function, we should pass down `$container` so the function has access to it.